contracts/DepositHandler.sol

```diff
@@ -71,6 +64,22 @@ contract DepositHandler is Vault {
     );
   }
 
+   /**
+   * @notice Add to the network `(_amountOrTokenId)` amount of a `(_color)` tokens
+   * or `(_amountOrTokenId)` token id if `(_color)` is NFT.
+   * @dev Token should be registered with the Bridge first.
+   * @param _owner Account to transfer tokens from
+   * @param _amountOrTokenId Amount (for ERC20) or token ID (for ERC721) to transfer
+   * @param _color Color of the token to deposit
+   */
+  function deposit(address _owner, uint256 _amountOrTokenId, uint16 _color) public {
+    _deposit(_owner, _amountOrTokenId, _color);
```

Comment:
Is this for backward-compatibility? Since in `_deposit` `owner` must now always be equal to `msg.sender`, right?

Reply:
i'm actually not aware of any case where the owner address is different from msg.sender. Just didn't want to break the interface with bridge-ui right now. But burner wallet would ideally use the new one.

Reply:
Makes sense to me 🤣